wallet/wallet.sol

```diff
@@ -52,6 +52,7 @@ contract multiowned {
     // constructor is given number of sigs required to do protected "onlymanyowners" transactions
     // as well as the selection of addresses capable of confirming them.
     function multiowned(address[] _owners, uint _required) {
+        if (_required > 1 && _owners.length < _required -1) throw;
```

Comment:
If we do such checks, we should probably also check for:
`if (_required == 0) throw;`
Also rewriting the current condition as `if (_required > _owners.length + 1)` would be more readable, I think.